onlinejudge_verify/main.py

```diff
@@ -178,14 +183,14 @@ def main(args: Optional[List[str]] = None) -> None:
     if parsed.subcommand == 'all':
         generate_gitignore()
         try:
-            subcommand_run(paths=[], jobs=parsed.jobs)
+            subcommand_run(paths=[], timeout=parsed.timeout, tle=parsed.tle, jobs=parsed.jobs)
         finally:
             # generate documents even if some tests failed
             subcommand_docs()
 
     elif parsed.subcommand == 'run':
         generate_gitignore()
-        subcommand_run(paths=parsed.path, jobs=parsed.jobs)
+        subcommand_run(paths=[], timeout=parsed.timeout, tle=parsed.tle, jobs=parsed.jobs)
```

Comment:
```suggestion
        subcommand_run(paths=parsed.path, timeout=parsed.timeout, tle=parsed.tle, jobs=parsed.jobs)
```

これ関係ないとこを間違えて変更しちゃってそう